filecoin/functions/piece-cid-report.js

```diff
@@ -28,27 +29,36 @@ async function pieceCidReport (event) {
   // @ts-expect-error can't figure out type of new
   const pieceRecord = unmarshall(records[0].new)
   const piece = Piece.fromString(pieceRecord.piece).link
+  const content = CID.parse(pieceRecord.link)
 
   const aggregateServiceConnection = getServiceConnection({
     did: aggregatorDid,
     url: aggregatorUrl
   })
+  const claimsServiceConnection = getServiceConnection({
+    did: aggregatorDid,
+    url: aggregatorUrl
+  })
   const issuer = getServiceSigner({
     privateKey
   })
-  const audience = aggregateServiceConnection.id
-  /** @type {import('@web3-storage/filecoin-client/types').InvocationConfig} */
-  const invocationConfig = {
-    issuer,
-    audience,
-    with: issuer.did(),
-  }
 
   const { ok, error } = await reportPieceCid({
     piece,
+    content,
     group: issuer.did(),
     aggregateServiceConnection,
-    invocationConfig
+    aggregateInvocationConfig: /** @type {import('@web3-storage/filecoin-client/types').InvocationConfig} */ ({
+      issuer,
+      audience: aggregateServiceConnection.id,
+      with: issuer.did(),
+    }),
+    claimsServiceConnection,
+    claimsInvocationConfig: /** @type {import('../types').ClaimsInvocationConfig} */ ({
+      issuer,
+      audience: claimsServiceConnection.id,
+      with: issuer.did(),
```

Comment:
You either need to use the claims service private key or delegate `assert/equals` to this issuer from the claims service (and include it as a proof).

Reply:
Added private key as secret as we talked